crates/biome_js_analyze/src/semantic_analyzers/nursery/use_number_properties.rs

```diff
@@ -0,0 +1,192 @@
+use crate::{semantic_services::Semantic, JsRuleAction};
+use biome_analyze::{
+    context::RuleContext, declare_rule, ActionCategory, FixKind, Rule, RuleDiagnostic,
+};
+use biome_console::markup;
+use biome_diagnostics::Applicability;
+use biome_js_factory::make;
+use biome_js_syntax::{global_identifier, AnyJsExpression, JsSyntaxKind, T};
+use biome_rowan::{AstNode, BatchMutationExt};
+
+declare_rule! {
+    /// Use `Number` properties instead of global ones.
+    ///
+    /// ECMAScript 2015 moved globals onto the `Number` constructor for consistency and to slightly improve them.
```

Comment:
I think the shorthand `ES2015` is more in use than `ECMAScript 2015`.

I could also add a note about the fact that the rule doesn't't report `isFinite` and `isNan`.

```suggestion
    /// _ES2015_ moved some globals into the `Number` namespace for consistency.
    ///
    /// The rule doesn't report the globals `isFinite` and `isNan` because they have a slightly different behabior to their corresponding `Number`'s properties `Number.isFinite` and `Number.isNan`.
    /// You can use the dedicated rules [noGlobalIsFinite](https://biomejs.dev/linter/rules/no-global-is-finite/) and  [noGlobalIsNan](https://biomejs.dev/linter/rules/no-global-is-nan/) to enforce the use of `Number.isFinite` and `Number.isNan`.
```

Reply:
fix: 99e0270

Reply:
@seitarof I think you missed the paragraph I added about `noGlobalIsFinite` and `noGlobalIsNan`.

Reply:
Sorry, I missed this. 
fix: e15f6d7